apps/desktop/src/views/SearchView/components/ConversationPanel/components/AssistantMessage.vue

```
@@ -14,8 +14,8 @@
 
                 <!-- 正常 AI 消息 -->
                 <template v-else>
-                    <!-- 推理过程显示（如果存在）-->
-                    <div v-if="message.reasoning" class="reasoning-section mb-4 w-full">
+                    <!-- 推理过程显示（兼容老消息：没有 reasoning parts 时）-->
+                    <div v-if="message.reasoning && !hasReasoningParts" class="reasoning-section mb-4 w-full">
```

ESLint

[error] Replace `·v-if="message.reasoning·&&·!hasReasoningParts"·class="reasoning-section·mb-4·w-full"` with `⏎························v-if="message.reasoning·&&·!hasReasoningParts"⏎························class="reasoning-section·mb-4·w-full"⏎····················`

(prettier/prettier)